pkg/sort/task_status_test.go

```diff
@@ -59,11 +60,12 @@ func TestStatusTmpl(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			console := consoleui.FallBackConsole{}
 			config := &info.ProviderConfig{
 				TaskStatusTMPL: tt.tmpl,
 			}
-			output, err := TaskStatusTmpl(tt.pr, console, config)
+			runs := params.New()
+			runs.Clients.ConsoleUI = consoleui.FallBackConsole{}
+			output, err := TaskStatusTmpl(tt.pr, tt.pr.Status.TaskRuns, runs, config)
```

Comment:
Is it possible to add ut for new function `GetStatusFromTaskStatusOrFromAsking`

Reply:
i tried, but it involves a lot of plumbing i'll get this later next week but let's merge this first for openshift release... (and it's only specific for tekton pipeline > 0.44)

Reply:
Sure!
Np

Reply:
done in #1140